Tests/CommonTests/SerializationTests.cs

```diff
@@ -78,8 +82,11 @@ public void SerializeRules()
     <severity>CRITICAL</severity>
     <cardinality>SINGLE</cardinality>
     <status>READY</status>
+    <type>CODE_SMELL</type>
     <tag>t1</tag>
     <tag>t2</tag>
+    <debtRemediationFunction>CONSTANT_ISSUE</debtRemediationFunction>
+    <debtRemediationFunctionOffset>15min</debtRemediationFunctionOffset>
```

Comment:
We don't currently generate template files files with tags or debt remediation data. However, this test checks that the serializer could correctly serialize that data. I'll add documentation to the readme explaining how to manually add these elements.